Samples/Tutorial Samples/WinMLPIXSample/WinMLPIXSample/WinMLHelper.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "pch.h"
+
+#ifndef WINMLHELPER_H
+#define WINMLHELPER_H
+
+using namespace winrt;
+using namespace winrt::Windows::AI::MachineLearning;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
+using namespace winrt::Windows::Graphics::Imaging;
+using namespace winrt::Windows::Media;
+using namespace winrt::Windows::Storage;
+using namespace std;
+
+// Global variables
+static hstring modelPath = L".\\Assets\\SqueezeNet.onnx";
+static string deviceName = "default";
+static hstring imagePath = L"\\Assets\\kitten_224.png";

```

Comment:
there's an error case in winmlhelper.cpp that says if the image loading fails, make sure you are using fully qualified paths. but this looks like a relative path. can we make this consistent?

Reply:
In the method 'LoadImageFile' there is a piece of code to obtain the full path with GetCurrentDirectory, personally I feel we should try to get the full path this way programmatically for all our samples. What do you think?

Reply:
With that said, hardcoding file path does seem less ideal, many other samples use the approach to get file path as input arguments. I can adopt to that as well.

Reply:
yes, let's do it programmatically. I thought we were being inconsistent, but it sounds like we always get the fully qualified path programmatically.